### PR TITLE
Human pose refactor

### PR DESCRIPTION
**Pull request by hobinjk-ptc**
Highly recommend viewing this with ?w=1 on the diff url

- Refactors out analysis-related functions in draw.js to be a modular class that can be ripped out later
- Rewrites every humanPose/* file as an es6 module that is re-exported by humanPose/index.js as realityEditor.humanPose.*

This rewrite and re-export is a pretty good model for how we can migrate the entire user interface to es6 modules which will in turn enable load time improvements through better tooling

### CODE REVIEW

**Review by dangond-ptc — APPROVED** [review]
Very nice to have more and more things available as imports!

**Comment by hobinjk-ptc** [comment]
When everything's an import and we can flip the switch to bring the user interface from 150 script requests to 1 it's going to be so good

File: src/humanPose/draw.js [hunk 1]
```diff
@@ -1,430 +1,456 @@
...
+/**
+ * Renders COCO-pose keypoints
+ */
+class HumanPoseRenderer {
+    /**
+     * @param {string} id - Unique identifier of human pose being rendered
+     * @param {THREE.Object3D} historyLineContainer - THREE container for
+     *                         history line meshes
+     * @param {THREE.Object3D} historyCloneContainer - THREE container for
+     *                         history clone meshes
```
Comment (benptc) [inline src/humanPose/draw.js:19]: nit: looks like the jsdoc params can be removed

**Review by benptc — APPROVED** [review]
Great to see a template for refactoring, plus the updates to humanPose/draw!

**Pull request merged**